src/c++/perf_analyzer/client_backend/tensorflow_serving/tfserve_grpc_client.cc

```diff
@@ -482,6 +484,9 @@ GrpcClient::PopulateInputData(
 {
   if (input->Datatype() == "FP16") {
     RETURN_IF_CB_ERROR(PopulateHalfVal(input_tensor_proto));
+  } else if (input->Datatype() == "BF16") {
+    return Error(
+        "BF16 datatype not currently supported for populating input data.");
```

Comment:
TF(S) doesn't specify how to represent BF16 in TensorProto? Or we are just not supporting it for now?

Reply:
Not supporting it for now. Would need to implement some kind of `PopulateBF16Val()` along with implementing TF backend support. I don't believe there is an issue on the TF side, but haven't looked deeply into it yet.

Reply:
We should file a JIRA ticket to track that. @tgerdesnv should the ticket be filed on tools team?

Reply:
@GuanLuo if it is a client issue, then yes please file it in the tools board